### PR TITLE
added explaination for "number of clusters"

### LINKED ISSUE #283: add explaination to color mapping module control
_Relevant file:_ https://github.com/darktable-org/dtdocs/blob/259a86bf72e40c329de283ab50c37db0236c3d33/content/module-reference/processing-modules/color-mapping.md
_Relevant section:_ number of clusters : The number of color clusters to use. If you change this parameter all collected color clusters are reset and must be re-acquired.

Reading the documentation, it's good to learn what effects changing the control has. But honestly by reading the section I have no clue about what the control actually does. What does a change do to the module functionality? I believe it would be good to add a sentence like "Increase the number of color clusters to _achieve A_ or decrease the number to _achieve B_." to add value to the documentation.

### PR DESCRIPTION
**Pull request by sjjh**
Enriched the explanation of the "number of clusters" control to give an idea when and how to change the parameter. Closes #283

### CODE REVIEW

**Comment by elstoc** [comment]
@matt-maguire please can you review - are you happy with this?

File: content/module-reference/processing-modules/color-mapping.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ acquire as source/target
 : Press these buttons to generate color clusters for the source and target image, respectively. The processing takes a few seconds during which the GUI will be unresponsive.
 
 number of clusters
-: The number of color clusters to use. If you change this parameter all collected color clusters are reset and must be re-acquired.
+: The number of color clusters to use. It should correlate to the number of dominant colors in the image. The clusters are aquired through random sampling, so if you add more clusters in a complex image, there is bigger chance that the clusters picked out through chance are not appropriate. If you change this parameter all collected color clusters are reset and must be re-acquired.
```
Comment (matt-maguire) [inline content/module-reference/processing-modules/color-mapping.md:28]: I would probably word it something like this:

The number of color clusters to use. It should roughly correlate to the main number of colours dominating the image. The clusters are required through a statistical sampling process to represent the colours occuring most commonly throughout the image. The random nature of the sampling process means that you might get varying results each time you aquire the clusters, especially if you have a larger number of clusters and/or a complex colour pallette in the image. If you change this parameter all the collected colour clusters are reset and must be re-acquired.

Comment (elstoc) [reply content/module-reference/processing-modules/color-mapping.md:28]: @matt-maguire: What is the "**main** number of colours"? Presumably you mean "The clusters are **acquired** through..."?

**Comment by sjjh** [comment]
so, how do we proceed here? :) Shall I do anything?

**Comment by elstoc** [comment]
I was waiting for a response from @matt-maguire to my question. I don't know the code so I'm trying to better understand how it works.

File: content/module-reference/processing-modules/color-mapping.md (hunk 1 above)
Comment (matt-maguire) [reply content/module-reference/processing-modules/color-mapping.md:28]: By "main number of colors" I mean "the number of dominant colors in the image"

Comment (elstoc) [reply content/module-reference/processing-modules/color-mapping.md:28]: Ok, so "the main number of colors dominating the image" was basically saying "the number of dominant colors dominating the image" so my alternative wording is ok?

**Comment by elstoc** [comment]
I pushed a commit with my suggested wording above. Shout if you think it needs further changes.

**Pull request closed without merging**